hack/verify-shellcheck.sh

```diff
@@ -66,16 +66,20 @@ remove_container () {
 # ensure we're linting the k8s source tree
 cd "${KUBE_ROOT}"
 
-# find all shell scripts excluding ./_*, ./.git/*, ./vendor*,
-# and anything git-ignored
+# Find all shell scripts excluding ./_*, ./.git/*, ./vendor*, ./third_party/*
+# except ./third_party/forked/*, and anything git-ignored.
```

Comment:
the double negatives here are confusing especially with the line split, let's try to rephrase this a little 🤔

Reply:
maybe:
```bash
# Find all shell scripts excluding:
# - anything git-ignored
# (no need to lint untracked files)
# - ./_* 
# (no need to lint output directories)
# - ./.git/* 
# (we only want to lint the source tree, not the git object store)
# - ./vendor* 
# (vendored code should be fixed upstream instead)
# - ./third_party/*, but re-include ./third_party/forked/* 
# (only code we forked should be linted and fixed)
```
and then drop the segment below?

Reply:
Updated with slightly modified version of your suggestion
```
# Find all shell scripts excluding:
# - Anything git-ignored - No need to lint untracked files.
# - ./_* - No need to lint output directories.
# - ./.git/* - Ignore anything in the git object store.
# - ./vendor* - Vendored code should be fixed upstream instead.
# - ./third_party/*, but re-include ./third_party/forked/*  - only code we
#    forked should be linted and fixed.
```